scss/components/navigation/_navigation.scss

```diff
@@ -12,7 +12,7 @@
 $nav-active-item-shadow: inset 0 -3px 0 0 rgba(249, 176, 2, 1);
 
 // Navigation styles
-.navigation {
+.navigation__main-menu {
```

Comment:
I changed this to be more specific because it was impossible to use the main-menu in a menu block anywhere else on the page than in the header. The block in the header uses a block class entry to get the styles.